tools/e2e/setup.mjs

```diff
@@ -88,26 +95,21 @@ export const startRegistry = async () => {
  * }
  */
 export const setup = async () => {
-  const { storage, url } = await startRegistry()
+  const { url } = await startRegistry()
   const workspace = await mkdtemp(`${tmpdir()}${sep}e2e-test-`)
 
-  const npmrc = join(homedir(), '.npmrc')
+  const npmrc = fileURLToPath(new URL('../../.npmrc', import.meta.url))
   const registryWithAuth = `//${url.hostname}:${url.port}/:_authToken=dummy`
   let backupNpmrc
 
   /** Cleans up everything */
   const cleanup = async () => {
-    // restore ~/.npmrc
```

Comment:
As we modify the npmrc file we need to clean it up again after everything was done or an error occured

Reply:
see other comment